PagingSample/app/src/main/java/paging/android/example/com/pagingsample/MainActivity.kt

```diff
@@ -61,9 +61,17 @@ class MainActivity : AppCompatActivity() {
     private fun initSwipeToDelete() {
         ItemTouchHelper(object : ItemTouchHelper.Callback() {
             // enable the items to swipe to the left or right
-            override fun getMovementFlags(recyclerView: RecyclerView,
-                                          viewHolder: RecyclerView.ViewHolder): Int =
-                makeMovementFlags(0, ItemTouchHelper.LEFT or ItemTouchHelper.RIGHT)
+            override fun getMovementFlags(
+                recyclerView: RecyclerView,
+                viewHolder: RecyclerView.ViewHolder
+            ): Int {
+                val cheeseViewHolder = viewHolder as CheeseViewHolder
+                return if (cheeseViewHolder.cheese != null) {
```

Comment:
simplify these 2 lines as:

`return if (viewHolder is CheeseViewHolder.Item)`

Reply:
Actually we only have one viewholder for both :|